assets/tests/components/propertiesPanel/minischedule.test.tsx

```diff
@@ -23,11 +23,6 @@ jest.mock("../../../src/hooks/useMinischedule", () => ({
   useMinischeduleBlock: jest.fn(),
 }))
 
-jest.mock("../../../src/laboratoryFeatures", () => ({
```

Comment:
**thought:** There were other changes in `minischedules.tsx`, but they didn't require test changes. I guess there's no test that icon directions match the ladder direction. Should there be?

Reply:
Okay, turns out it's actually kind of difficult - we don't have good ways of testing things related to the actual contents of the SVGs, and we don't have much in the way of accessibility metadata to make it easy to select said elements in React Testing Library. The alternative is a snapshot test but those don't actually include the rendered contents of the SVGs and there isn't other information in existing classnames or anything to differentiate the icon types in a snapshot.